client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTableFooter/GroupedInvestigations/GroupedInvestigations.tsx

```diff
@@ -13,16 +13,18 @@ import GroupedInvestigationsFields from './GroupedInvestigationsForm/GroupedInve
 const title = 'קיבוץ חקירות'
 
 const GroupedInvestigations: React.FC<Props> = ({ invetigationsToGroup, open, onClose, fetchTableData }: Props) => {
-
+    const groupedInvestigation: InvestigationTableRow | undefined = invetigationsToGroup
+        .find((investigation: InvestigationTableRow) => investigation.groupId !== null);
```

Comment:
maybe put it inside useMemo and return boolean instead of array and change the name properly